src/components/IdEntry.vue

```diff
@@ -295,14 +315,21 @@ export default {
     /** This method is called whenever "+" button is clicked. It authenticates the most recent typed ID.
      * @param {String} userID - most recent ID
      */
-    async authenticateSRN(userID) {
+    async authenticateID(userID) {
       this.isLoading = true;
-      let userValidationResponse = await validateSRN(userID, this.validateCount);
+      let userValidationResponse = await validateID(
+        userID,
+        this.validateCount,
+        this.dataSourceObject["name"],
+        this.dataSourceObject["column"]
+      );
       this.isCurrentUserValid = userValidationResponse.isCurrentUserValid;
       this.validateCount = userValidationResponse.validateCount;
-      this.invalidLoginMessage = userValidationResponse.invalidLoginMessage;
       this.isLoading = false;
 
+      if (this.validateCount == 1) {
```

Comment:
i thought we had decided to wrap such conditions under methods/computed properties with meaningful names, right?